charts/shoot-core/charts/monitoring/charts/blackbox-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: apiserver-connectivity-check
+  namespace: kube-system
+  labels:
+    garden.sapcloud.io/role: system-component
+    component: apiserver-connectivity-check
+    addonmanager.kubernetes.io/mode: Reconcile
+    origin: gardener
+spec:
+  revisionHistoryLimit: 0
+  replicas: 1
+  selector:
+    matchLabels:
+      component: apiserver-connectivity-check
+  template:
+    metadata:
+      annotations:
+        checksum/configmap-gardener-controller-manager-config: {{ include (print $.Template.BasePath "/config.yaml") . | sha256sum }}
```

Comment:
misleading annotation name, should be `checksum/configmap-blackbox-exporter-config`

Reply:
done